Tests are breaking when source data changes slightly

The assertions
https://github.com/sledilnik/py-cepimose/blob/5f4ff4f2cbd9ab9447671026f2fecdf779c6c43c/test/test.py#L95-L96

is [failing the Test run](https://github.com/sledilnik/py-cepimose/runs/2284923003?check_suite_focus=true#step:5:34) with:
> AssertionError: 48799 != 48798

because the data in the dashboard changed slightly.

Tests should allow for some tolerance.

--- a/test/test.py
+++ b/test/test.py
@@ -17,8 +17,8 @@ class CepimoseTestCase(unittest.TestCase):
 
         def assertRow(row, expected_date, expected_first, expected_second):
             self.assertEqual(row.date, expected_date)
-            self.assertEqual(row.first_dose, expected_first)
-            self.assertEqual(row.second_dose, expected_second)
+            self.assertAlmostEqual(row.first_dose, expected_first, delta=10)
+            self.assertAlmostEqual(row.second_dose, expected_second, delta=5)
 
         assertRow(data[9], datetime.datetime(2021, 1, 5), 15711, 0)
         assertRow(data[22], datetime.datetime(2021, 1, 18), 48710, 315)
@@ -89,8 +89,8 @@ class CepimoseTestCase(unittest.TestCase):
 
         def assertRow(row, expected_date, expected_supp, expected_used):
             self.assertEqual(row.date, expected_date)
-            self.assertEqual(row.supplied, expected_supp)
-            self.assertEqual(row.used, expected_used)
+            self.assertAlmostEqual(row.supplied, expected_supp, delta=10)
+            self.assertAlmostEqual(row.used, expected_used, delta=10)
 
         assertRow(data[9], datetime.datetime(2021, 1, 4), 39780, 13248)
         assertRow(data[22], datetime.datetime(2021, 1, 17), 60870, 48798)

